airflow/triggers/temporal.py

```diff
@@ -54,7 +54,7 @@ async def run(self):
         unexpectedly, or handles a DST change poorly.
         """
         # Sleep an hour at a time while it's more than 2 hours away
-        while (self.moment - timezone.utcnow()).total_hours() > 2:
+        while ((self.moment - timezone.utcnow()).total_seconds() / 3600) > 2:
```

Comment:
Why make this change? I'm curious.

Reply:
The test fails on AttributeError complaining that there’s no total_hours on a timedelta

Reply:
Strange... when I ran it locally it was a `pendulum.Period` rather than a `datetime.timedelta`. Let me try it locally again.

Reply:
Hah, looks like a change on `main` in the last few days means it's now a timedelta, odd! I would suggest moving the `/3600` over to become a `2*3600` though, for readability and less nesting?

Reply:
Cool!